docs/layer_reference/recurrent.rst

```diff
@@ -11,13 +11,34 @@ Choice Layer
     :members:
     :undoc-members:
 
-Decision Layer
---------------
+Choice Get Beam Scores Layer
+----------------------------
+
+.. autoclass:: returnn.tf.layers.rec.ChoiceGetBeamScoresLayer
+    :members:
+    :undoc-members:
```

Comment:
This is a case which is used by some, but I'm not sure whether it might be deprecated at some point, because I think all these `GetXYLayer` types are ugly, and only like this because we don't have a nicer syntax currently. E.g. we had some `GetSubLayer` layer before the syntax `subnet/sublayer` was introduce.

Reply:
But as long as there is no alternative, it should be included. We can remove it as soon as there is something else.

Reply:
But then it should at least have a big disclaimer that this is probably going to be deprecated at some point.